doc/vdo.rst

```diff
@@ -251,7 +251,16 @@ The messages are:
 		by the vdostats userspace program to interpret the output
 		buffer.
 
-        dump:
+	config:
+		Outputs useful vdo configuration information in the form:
+		{ version: 1, physicalSize: <bytes>, logicalSize: <bytes>,
+		  slabSize: <bytes>, index: { memorySize: <gb>,
+		  isSparse: <0 or 1>, },
+		}
+
+{ version: 1, physicalSize, logical }
+
```

Comment:
Other message documentation doesn't provide details about their output. I think that is better because we won't have to update documentation every time we make a minor change to the message output. Also, I am not sure what purpose line 261 serves, is it here intentionally?

Reply:
Ok Ill remove the output. I thought it might be useful to clients so they know how to parse it.

Reply:
alright i'll add some more description